lib/boolean.js

```diff
@@ -41,6 +54,50 @@ internals.Boolean = class extends Any {
         result.errors = (typeof result.value === 'boolean') ? null : this.createError('boolean.base', null, state, options);
         return result;
     }
+
+    truthy(truthyValue) {
+
+        let truthySet = [];
+
+        if (Array.isArray(truthyValue)) {
+            truthySet = truthyValue.map((value) => {
+
+                Hoek.assert(typeof value === 'string', 'truthy value must be a string');
+
+                return value.toLowerCase();
+            });
+        }
+        else {
```

Comment:
This is actually inconsistent with how we do it everywhere else, see https://github.com/hapijs/joi/blob/master/lib/any.js#L235.